drizzlepac/haputils/processing_utils.py

```diff
@@ -161,8 +161,8 @@ def refine_product_headers(product, total_obj_list):
     phdu['numexp'] = len(input_exposures)
 
     # Convert dates to ISO format
-    phdu['date-beg'] = (Time(phdu['expstart'], format='mjd').iso, "Starting Date and Time")
-    phdu['date-end'] = (Time(phdu['expend'], format='mjd').iso, "Ending Date and Time")
+    phdu['date-beg'] = (Time(phdu['expstart'], format='mjd').iso.replace(' ', 'T'), "Starting Date and Time")
+    phdu['date-end'] = (Time(phdu['expend'], format='mjd').iso.replace(' ', 'T'), "Ending Date and Time")
```

Comment:
```suggestion
    phdu['date-beg'] = (Time(phdu['expstart'], format='mjd').isot, "Starting Date and Time")
    phdu['date-end'] = (Time(phdu['expend'], format='mjd').isot, "Ending Date and Time")
```

See https://docs.astropy.org/en/stable/api/astropy.time.TimeISOT.html#astropy.time.TimeISOT